extensions/spatial.sigmf-ext.md

```diff
@@ -0,0 +1,362 @@
+# The `spatial` SigMF Extension Namespace v0.0.2
+
+This document defines the `spatial` extension namespace for the Signal Metadata
+Format (SigMF) specification. This extension namespace contains objects to help
+store information about spatially diverse data, specifically information applied
+to directional antennas and multichannel phase-coherent datasets used for signal
+direction of arrival and beamforming.
+
+Multichannel datasets can be stored in SigMF Collections or as multichannel
+interleaved data depending on the application, Collections are RECOMMENDED due
+to the easier processing and better application support.
+
+The `spatial` extension makes use of cartesian coordinates to define array
+geometry, and spherical coordinates for reporting bearings. This, coupled with
+the various methods for defining boresights and bearing references can become
+complicated rapidly so the union of these two cordinate reference systems (CRS)
+is illustrated in Figure 1 below. The ISO 80000-2:2019 compliant cartesian and
+spherical systems have a specific relationship with the reported azimuth and
+elevation, which use the conventional geospatial definitions (degrees east of
+north, and degrees above horizon respectively).
+
+![SigMF Spatial Coordinate Reference System](spatial_crs.png)
+
+**Figure 1 - SigMF Spatial Extension Coordinate Reference Systems**
+
+As shown in Figure 1, the boresight of the aperture is defined as being in the
+direction of the positive X-axis. Based on this, as an example, a horizontal
+uniform linear array aperture would be defined along the Y-axis (see examples
+section below).
+
+## 0 Datatypes
```

Comment:
I certainly am a believer in zero-indexed arrays, but I would think that `1` would be a more intuitive starting point here.  Does numbering these sections contribute to the presentation? Like, does it make it more clear in some way?